site/blog/2023/11/identity-synchronization.md

```diff
@@ -0,0 +1,54 @@
+---
+type: article
```

Comment:
What is it used for?

Reply:
It was a mistake that i left in the code from previous implementation idea.